server/pbenchinacan/pod.yml

```diff
@@ -21,20 +12,22 @@
 # with the following environment variables defined:
 #
 #  PB_POD_NAME - the name to be used for the pod
-#  PB_INTERNAL_CONTAINER_REG - the FQDN of the container registry
-#  PB_SERVER_IMAGE_TAG - the tag for all three container images
+#  PB_CONTAINER_REG - the FQDN of the container registry
+#  PB_REPO_NAME - the image repository in the container registry
+#  PB_<ELASTICSERACH|POSTGRESQL|SERVER>_IMAGE_TAG - the tag container image
+#  PB_<ELASTICSERACH|POSTGRESQL|SERVER>_IMAGE_NAME - the image name to use
+#  PB_<ELASTICSERACH|POSTGRESQL|SERVER>_IMAGE_PULL_POLICY - pull policy to use
+#  PB_DASHBOARD_DIR - the directory to mount into the pbench server container
```

Comment:
**Thirteen** variables?  That pretty much makes this unusable without wrapping it in a script.  😢

Do we really need to parameterize all the individual components of each of the three (soon to be _four_) containers' names?  (And, do we really need to give each of them independent pull policies?)

(And, you should fix the spelling of `ELASTICSEARCH`.)

I realize that you are focusing on being able to deploy a staging server and on trying to converge the various pod YAML files, but how are you expecting a team member to pull and run the pod?  My intention was for there to be a single, reasonably simple command which would do that, and I think we've lost that with this change.

Reply:
Addressed in PR #3112.